Samples/DynamicDependencies/Basic/cpp-console-unpackaged/DynamicDependenciesBasic/DynamicDependenciesBasic.cpp

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright(c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// DynamicDependenciesBasic.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+#include <windows.h>
+#include <MddBootstrap.h>
+
+// Used to demo usage of a Windows App SDK api
+#include "winrt\Microsoft.Windows.System.Power.h"
+
+// Reference cppwinrt nuget in order to use WinRT apis:
+using namespace winrt;
+using namespace winrt::Microsoft::Windows::System::Power;
+
+// Constants for dyanmic dependencies lookup
+// The following code is for 1.0 Preview 2. 
+// Preview tag not needed for GA release
+// PCWSTR versionTag{ L"" };
+const UINT32 majorMinorVersion{ 0x00010000 };
+PCWSTR versionTag{ L"preview2" };
+
+int main()
+{
+    // Initialize values for the call to the Dynamic Dependencies bootstrapper
+    const PACKAGE_VERSION minVersion{};
+    const HRESULT hr{ MddBootstrapInitialize(majorMinorVersion, versionTag, minVersion) };
```

Comment:
use wil error handling macros, `THROW_IF_FAILED()` or `RETURN_IF_FAILED`. note is it is a little unusual for main to return an HRESULT.